src/test/groovy/org/prebid/server/functional/tests/AmpSpec.groovy

```diff
@@ -180,4 +183,79 @@ class AmpSpec extends BaseSpec {
         assert bidderRequest.imp[0]?.banner?.format[0]?.weight == ampStoredRequest.imp[0].banner.format[0].weight
         assert bidderRequest.regs?.gdpr == ampStoredRequest.regs.gdpr
     }
+
+    def "PBS should pass addtl_consent to user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers"() {
+        given: "Default amp request with addtlConsent"
+        def randomAddtlConsent = PBSUtils.randomString
+        def ampRequest = AmpRequest.defaultAmpRequest.tap {
+            addtlConsent = randomAddtlConsent
+        }
+
+        and: "Save storedRequest into DB"
+        def ampStoredRequest = BidRequest.defaultBidRequest.tap {
+            user = new User(ext: new UserExt(
+                    consentedProvidersSettingsKebabCase: new ConsentedProvidersSettings(consentedProviders: PBSUtils.randomString),
+                    consentedProvidersSettings: new ConsentedProvidersSettings(consentedProviders: PBSUtils.randomString)))
+        }
+        def storedRequest = StoredRequest.getStoredRequest(ampRequest, ampStoredRequest)
+        storedRequestDao.save(storedRequest)
+
+        when: "PBS processes amp request"
+        defaultPbsService.sendAmpRequest(ampRequest)
+
+        then: "Bidder request should contain addtl consent"
+        def bidderRequest = bidder.getBidderRequest(ampStoredRequest.id)
+        assert bidderRequest.user.ext.consentedProvidersSettingsKebabCase.consentedProviders == randomAddtlConsent
+        assert bidderRequest.user.ext.consentedProvidersSettings.consentedProviders == randomAddtlConsent
+    }
+
+    def "PBS should leave requested user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers as is"() {
```

Comment:
PBS should process original user.ext.{consented_providers_settings/ConsentedProvidersSettings}.consented_providers when ampRequest doesn't contain addtl_consent